core/src/test/java/org/testcontainers/images/RemoteDockerImageTest.java

```diff
@@ -0,0 +1,47 @@
+package org.testcontainers.images;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.testcontainers.utility.Base58;
+
+import java.util.concurrent.CompletableFuture;
+
+public class RemoteDockerImageTest {
+
+    @Test
+    public void toStringContainsOnlyImageName() {
```

Comment:
This method name came to be back when there was a choice between imageName and imageNameFuture.  Maybe a better name now is `toStringWithCompletedImageName` ?